src/cms/apps.py

```diff
@@ -23,6 +23,9 @@ class CmsConfig(AppConfig):
     :type name: str
     """
 
+    # Monkeypatch module path of AppConfig to allow Intersphinx mapping
+    AppConfig.__module__ = "django.apps"
```

Comment:
@svenseeberg Sorry, I missed this when outsourcing the other module path fixes to [sphinxcontrib-django2](https://github.com/timoludwig/sphinxcontrib-django2/blob/e3acb773a5bec7c3243a6f5204c4a2364c7a9878/sphinxcontrib_django2/docstrings/patches.py#L17). I will put this patch in the extension as well, so it only affects documentation builds (even though I'm pretty sure:tm: it doesn't have side-effects).

Reply:
Done: https://github.com/timoludwig/sphinxcontrib-django2/commit/12092a94da0d493f4849ca5515d1995daf46eb4e